frontend/src/schema/feedSchema.json

```diff
@@ -0,0 +1,91 @@
+{
+	"schema": {
+		"type": "object",
+		"required": ["name", "description", "search", "listeners"],
+		"properties": {
+			"name": {
+				"type": "string",
+				"title": "Name"
+			},
+			"description": {
+				"type": "string",
+				"title": "Description"
+			},
+			"search": {
+				"type": "object",
+				"title": "Match criteria",
+				"required": ["mode", "criteria"],
+				"description": "Fill out the match criteria to enable the selected extractor(s) on those specific files/datasets that match the criteria",
+				"properties": {
+					"mode": {
+						"type": "string",
+						"title": "Mode of match criterion",
+						"enum": ["Or", "And"]
+					},
+					"criteria": {
+						"type": "array",
+						"title": "",
+						"items": {
+							"type": "object",
+							"required": ["field", "operator", "value"],
+							"properties": {
+								"field": {
+									"type": "string",
+									"title": "Field",
+									"examples": [
+										"content_type_main",
+										"content_type",
+										"size",
+										"creator"
+									]
+								},
+								"operator": {
+									"type": "string",
+									"title": "Operator",
+									"enum": ["==", "<", ">", "!="]
```

Comment:
It would be nice to add "regex" support for the future so we can match even fine gridded. Is there a separate task for that?

Reply:
For now, I think we wanted it as a fixed list of options , as per discussion with @lmarini
However I created a separate task #1102